src/client/pages/Welcome.tsx

```diff
@@ -8,14 +8,25 @@ import {
 } from '@/client/layouts/shared/Consents';
 import { getAutoRow, gridItemColumnConsents } from '@/client/styles/Grid';
 import { CONSENTS_PAGES } from '@/client/models/ConsentsPages';
+import { sendOphanReferrerEvent } from '../lib/ophan';
```

Comment:
```suggestion
import { sendOphanReferrerEvent } from '@/client/lib/ophan';
```

small style change 😆